YalantisTest/Sources/Screens/MagicBall/BallModel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BallModel.swift
+//  YalantisTest
+//
+//  Created by Vitaliy Yefimchuk on 14.11.2021.
+//
+
+import Foundation
+
+class BallModel {
```

Comment:
You can set as `final` class and code will compile faster. Will change dynamic table dispatching to static(if hasn't subclasses).